src/components/loadable-item.cjsx

```diff
@@ -31,15 +31,16 @@ module.exports = React.createClass
   componentDidUpdate: (oldProps) -> @reload(oldProps)
 
   reload: (oldProps) ->
-    {id, store, load, actions, options} = @props
+    {id, store, load, actions, options, isLoading} = @props
     return unless id?
 
     # Skip reloading if all the props are the same (the case in the Calendar for some reason)
     return if oldProps.id is id and oldProps.store is store and oldProps.actions is actions and
       oldProps.load is load and _.isEqual(oldProps.options, options)
 
     load ?= actions.load
-    unless store.isNew(id, options)
+    isLoading ?= store.isLoading
+    unless store.isNew(id, options) and not isLoading?(id, options)
```

Comment:
hmmm...i see how this is the same but would `unless isLoading?(id, options) or store.isNew(id, options)` be more readable?